Morcoso_Daisy/Countries of the world/index.html

```diff
@@ -0,0 +1,22 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Country Info Finder</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <h1>Country Info Finder</h1>
+  <div id="search_container">
+    <input type="text" id="countryInput" placeholder="Enter country name">
+    <button id="searchButton">Search</button>
```

Comment:
Your ID's in the codes must be written in snake case. It should be all lowercase letters that are separated by an underscore.

```suggestion
    <input type="text" id="country_input" placeholder="Enter country name">
```

For reference follow the guidelines:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5